### PR TITLE
Develop1

### PR DESCRIPTION
**Pull request by Maneesha285**

### CODE REVIEW

File: .idea/main.txt [hunk 1]
```diff
@@ -1 +1 @@
-sunday
+monday tuesday
```
Comment (Nani0218) [inline .idea/main.txt:1]: First letter should be captal

**Review by Nani0218 — CHANGES_REQUESTED** [review]

File: .idea/main.txt (hunk 1 above)
Comment (Maneesha285) [reply .idea/main.txt:1]: okay

Comment (Nani0218) [inline .idea/main.txt:1]: no changes

Comment (Nani0218) [inline .idea/main.txt:1]: no changes

Comment (Nani0218) [reply .idea/main.txt:1]: pls changes

**Pull request merged**